### ISSUE TITLE
Reactor stabilizer and Open Computers Adapter kick

### ISSUE THREAD

**Issue by Joshi120** [brandon3055/Draconic-Evolution#331]
## Description

Hello :) i wasn't sure on which mod page i should report it but i hope it is fine here so to the issue When ever you place a reactor Stabilizer on top of a Opencomputers Adapter you get kicked from the server this only happens in MP it works just fine in SP

## Base information
* Minecraft version:1.7.10
* Minecraft Forge version: Forge Mod Loader version 7.99.39.1566
* Draconic Evolution version: 1.0.2

## Mod List or Mod Pack (include version)
...  Ferret Business 0.1.3
list other mods being used
...

## Crash report
```
`[02:23:37] [Server thread/WARN]: Failed to handle packet for /127.0.0.1:54831
openperipheral.adapter.composed.ComposedMethodsFactory$InvalidClassException
        at openperipheral.adapter.composed.ComposedMethodsFactory.getAdaptedClass(ComposedMethodsFactory.java:46) ~[ComposedMethodsFactory.class:?]
        at openperipheral.interfaces.oc.providers.DriverOpenPeripheral.shouldProvide(DriverOpenPeripheral.java:37) ~[DriverOpenPeripheral.class:?]
        at openperipheral.interfaces.oc.providers.DriverOpenPeripheral.worksWith(DriverOpenPeripheral.java:28) ~[DriverOpenPeripheral.class:?]
        at li.cil.oc.server.driver.Registry$$anonfun$2.apply(Registry.scala:70) ~[Registry$$anonfun$2.class:?]
        at li.cil.oc.server.driver.Registry$$anonfun$2.apply(Registry.scala:70) ~[Registry$$anonfun$2.class:?]
        at scala.collection.TraversableLike$$anonfun$filterImpl$1.apply(TraversableLike.scala:259) ~[scala-library-2.11.1.jar:?]
        at scala.collection.mutable.ResizableArray$class.foreach(ResizableArray.scala:59) ~[scala-library-2.11.1.jar:?]
        at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48) ~[scala-library-2.11.1.jar:?]
        at scala.collection.TraversableLike$class.filterImpl(TraversableLike.scala:258) ~[scala-library-2.11.1.jar:?]
        at scala.collection.TraversableLike$class.filter(TraversableLike.scala:270) ~[scala-library-2.11.1.jar:?]
        at scala.collection.AbstractTraversable.filter(Traversable.scala:104) ~[scala-library-2.11.1.jar:?]
        at li.cil.oc.server.driver.Registry$.driverFor(Registry.scala:70) ~[Registry$.class:?]
        at li.cil.oc.api.Driver.driverFor(Driver.java:96) ~[Driver.class:?]
        at li.cil.oc.common.tileentity.Adapter.neighborChanged(Adapter.scala:58) ~[Adapter.class:?]
        at li.cil.oc.common.block.Adapter.onNeighborChange(Adapter.scala:42) ~[Adapter.class:?]
        at net.minecraft.world.World.func_147453_f(World.java:3875) ~[ahb.class:?]
        at net.minecraft.world.World.func_147455_a(World.java:2603) ~[ahb.class:?]
        at net.minecraft.world.chunk.Chunk.func_150806_e(Chunk.java:851) ~[apx.class:?]
        at net.minecraft.world.chunk.Chunk.func_150807_a(Chunk.java:660) ~[apx.class:?]
        at net.minecraft.world.World.func_147465_d(World.java:451) ~[ahb.class:?]
        at net.minecraft.item.ItemBlock.placeBlockAt(ItemBlock.java:206) ~[abh.class:?]
        at net.minecraft.item.ItemBlock.func_77648_a(ItemBlock.java:103) ~[abh.class:?]
        at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505) ~[ForgeHooks.class:?]
        at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128) ~[add.class:?]
        at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389) ~[mx.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556) ~[nh.class:?]
        at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60) ~[jo.class:?]
        at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9) ~[jo.class:?]
        at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212) ~[ej.class:?]
        at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165) [nc.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685) [li.class:?]`
```

**Comment by brandon3055** [comment]
I was not able to recreate this issue but i did have to update my OC api before testing which may have been the cause so let me know if it is still an issue after the next release.

**Comment by brandon3055** [comment]
@Joshi120 Can you please check if the issue is fixed or not in the latest version? (1.0.2f)

**Comment by Joshi120** [comment]
@brandon3055 I just updated to 1.0.2f  and it looks fixed :) didn't got kicked when placing the stabilizer.
